pkg/storage/storage.go

```diff
@@ -88,6 +92,16 @@ func (s *storageModule) Total(kind pkg.DeviceType) (uint64, error) {
 	return total, nil
 }
 
+// BrokenPools lists the broken storage pools that have been detected
+func (s *storageModule) BrokenPools() []pkg.BrokenPool {
+	return s.brokenPools
```

Comment:
shouldn't these be protected by a mutex  ?

Reply:
I don't think so since the storageModule fields are only written during initialization atm, and access only happens through zbus which should be after initialization. But I'll add it since that can't be defered from just the storage code anyway